12LessonTwelve/homework/index.html

```diff
@@ -5,6 +5,53 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <div class="container">
+    <header>
+      <h1>Best Selling Products</h1>
+    </header>
+    <section>
+      <article>
+        <div>
+          <div class="img">
+            <img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/185649/hoodie_1_front.jpg" alt="" />
+            <div class="amount"><b>$35.00</b></div>
+          </div>
+          <h3>Patient Ninja</h3>
+          <a href="#">add to cart</a>
+        </div>
+      </article>
+      <article>
+            <div>
+              <div class="img">
+                <img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/185649/hoodie_1_front.jpg" alt="" />
+                <div class="amount"><b>$35.00</b></div>
+              </div>
+              <h2>Happy Ninja</h2>
+              <a href="#">add to cart</a>
+            </div>
+      </article>
+          <article>
+            <div>
+              <div class="img">
+                <img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/185649/hoodie_1_front.jpg" alt="" />
+                <div class="sale"><span>sale</span></div>
+                <div class="price-change"><span>$15.00</span><b>$12.00</b></div>
```

Comment:
Why not make this class `amount` too? It would go a long way towards a more consistent layout.
Remember, multiple classes use the same `class=` and each class is separated by a space.
ie ```class="price-change amount another_class one-more-for-good-measure```